AlphaWallet/Tokens/ViewControllers/TokenInstanceActionViewController.swift

```diff
@@ -208,34 +209,49 @@ class TokenInstanceActionViewController: UIViewController, TokenVerifiableStatus
             }
 
             guard let (data, value) = transactionFunction.generateDataAndValue(withTokenId: tokenId, attributeAndValues: values, localRefs: strongSelf.tokenScriptRendererView.localRefs, server: strongSelf.server, session: strongSelf.session, keystore: strongSelf.keystore) else { return }
-            let eth = EtherNumberFormatter.full.string(from: BigInt(value))
-            let nativeCryptSymbol: String
-            switch strongSelf.server {
-            case .xDai:
-                nativeCryptSymbol = "xDAI"
-            case .artis_sigma1, .artis_tau1:
-                nativeCryptSymbol = "ATS"
-            case .rinkeby, .ropsten, .main, .custom, .callisto, .classic, .kovan, .sokol, .poa, .goerli:
-                nativeCryptSymbol = "ETH"
-            }
-            if let data = data {
+//            let eth = EtherNumberFormatter.full.string(from: BigInt(value))
```

Comment:
Can you remove the commented out code? We can get it back from version control next time.